### ISSUE TITLE
Uncaught TypeError: img.toPng is not a function

### ISSUE THREAD

**Issue by jonaprieto** [tlnagy/atom-markdown-image-assistant#16]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.28.1 x64
**Electron**: 2.0.4
**OS**: Mac OS X 10.13.5
**Thrown From**: [markdown-image-assistant](https://github.com/tlnagy/atom-markdown-image-assistant) package 0.2.0

### Stack Trace

Uncaught TypeError: img.toPng is not a function

```
At /Users/jonaprieto/.atom/packages/markdown-image-assistant/lib/main.coffee:89

TypeError: img.toPng is not a function
    at Object.handle_cp (/packages/markdown-image-assistant/lib/main.coffee:89:25)
    at /packages/markdown-image-assistant/lib/main.coffee:56:18
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:27:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:156:34)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:360:24)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:110:40)
```

### Commands

```
     -3:19.3.0 editor:paste-without-reformatting (input.hidden-input)
  2x -2:58.4.0 core:paste (div.line.cursor-line)
     -2:52.3.0 editor:paste-without-reformatting (input.hidden-input)
     -2:49 command-palette:toggle (input.hidden-input)
     -2:47.1.0 core:confirm (input.hidden-input)
     -2:47.1.0 settings-view:uninstall-packages (input.hidden-input)
     -2:31.5.0 core:confirm (input.hidden-input)
     -2:00.4.0 editor:move-to-first-character-of-line (input.hidden-input)
  6x -1:56.9.0 core:delete (input.hidden-input)
  3x -1:55.3.0 core:backspace (input.hidden-input)
     -1:20.1.0 core:save (input.hidden-input)
     -0:56.9.0 core:paste (input.hidden-input)
     -0:31.4.0 command-palette:toggle (a.stack-toggle)
  3x -0:28.7.0 core:backspace (input.hidden-input)
     -0:26.1.0 core:confirm (input.hidden-input)
     -0:26.1.0 settings-view:uninstall-packages (a.stack-toggle)
```

### Non-Core Packages

```
agda-mode 0.7.13 
date 2.1.0 
language-agda 1.0.3 
language-latex 1.1.1 
language-liquid 0.7.0 
latex 0.50.0 
latex-tree 0.3.0 
latextools 0.8.5 
markdown-image-assistant 0.2.0 
markdown-image-insertion 0.2.1 
markdown-img-paste 0.3.11 
markdown-writer 2.7.4 
```

**Comment by jonaprieto** [comment]
https://github.com/bigyuki/markdown-image-helper/issues/14 that's was the solution

**Comment by jonaprieto** [comment]
It will be closed by #17

**Comment by tlnagy** [comment]
Yup. I merged #15 and I'll push a new version here in a sec.
